### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by jazairi**
## Status
READY

#### What does this PR do?
Updates all app dependencies.

#### Todo:
- [x] Tests
- [ ] Documentation
- [ ] Stakeholder approval

#### Requires Database Migrations?
YES

#### Includes new or updated dependencies?
YES

### CODE REVIEW

**Comment by jazairi** [comment]
@JPrevost I just updated the gems that depfu found individually, but `bundle update` found 11 more minor and patch updates. Should I push those as well? The tests pass.

**Comment by JPrevost** [comment]
@jazairi Yeah it's totally cool to update as much as works. I'll show you my process for sanity checking this app in the PR build on Monday :)

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_02_27_190524) do
+ActiveRecord::Schema.define(version: 2018_02_27_185947) do
```
Comment (JPrevost) [inline db/schema.rb]: I'm curious if you know why this was needed? I'm a bit confused by it. I don't think it hurts anything but it also seems odd.

Comment (jazairi) [reply db/schema.rb]: Nope, I'm as confused as you are. Let's talk about it today.

Comment (jazairi) [reply db/schema.rb]: @JPrevost Am I clear to resolve this? My understanding is that we determined the timestamp is correct, despite its weirdness.

Comment (JPrevost) [reply db/schema.rb]: Yeah I think we're good. I'll resolve the convo. Thanks for following up to confirm.

**Pull request merged**